src/colp/components/agency_usetype_report.py

```diff
@@ -0,0 +1,78 @@
+import streamlit as st
+from src.constants import COLOR_SCHEME
+from abc import ABC
+import plotly.express as px
+
+
+class CountRecordsReport(ABC):
+    def __call__(self):
+        if self.data is None:
+            st.info(f"No Count of records by {self.X_axis_col}")
+            return
+        by_agency = ""
```

Comment:
think its maybe cleaner to split this section out into a separate function like section_title_text or something